starlite/handlers/http.py

```diff
@@ -312,7 +319,7 @@ class HTTPRouteHandler(BaseRouteHandler["HTTPRouteHandler"]):
     @validate_arguments(config={"arbitrary_types_allowed": True})
     def __init__(
         self,
-        path: Union[Optional[str], Optional[List[str]]] = None,
+        path: Optional[Union[str, Sequence[str]]] = None,
```

Comment:
The types changed here also need to be changed in the docstring, but I can't comment on that